src/app/app.module.ts

```diff
@@ -60,6 +60,7 @@ import { PartnersComponent } from './components/home/partners/partners.component
 import { RulesComponent } from './components/home/rules/rules.component';
 import { TestimonialsComponent } from './components/home/testimonials/testimonials.component';
 import { FeaturedChallengesComponent } from './components/home/featured-challenges/featured-challenges.component';
+import { Ng2CarouselamosModule } from 'ng2-carouselamos';
```

Comment:
If you changed the versions of @angular module to make `ng2-carouselamos` compatible for the project, then please let us know whether it will work on Latest Angular version or not (Because near future, surely this project will be upgraded to the latest Angular version, at that time this module should not block the project update )

Reply:
Yes it will work with the updated versions :)